### PR TITLE
Add appendToFile

### PR DESCRIPTION
**Pull request by niccord**
Fix issue #1

### CODE REVIEW

**Comment by Eomm** [comment]
Hi,
thank you for the contribution!

Could you add some tests for that function and update the readme?

File: append.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello world - this is some added text
```
Comment (Eomm) [inline append.txt:1]: This file, if needed, should be moved on `test/assets/append` directory

File: test/file-utils-easy.test.js [hunk 2]
```diff
@@ -35,6 +35,32 @@ describe('write tests', () => {
   });
 });
 
+describe('append tests', () => {
+  it('create with append relative', () => {
+    const fileContent = 'Hello world';
+    const filePath = 'append.txt';
+    if (fue.existFile(filePath)) {
```
Comment (Eomm) [inline test/file-utils-easy.test.js]: This function returns a Promise, so the behaviour is not predictable

File: test/file-utils-easy.test.js [hunk 3]
```diff
@@ -35,6 +35,32 @@ describe('write tests', () => {
...
+      fue.deleteFile(filePath);
```
Comment (Eomm) [inline test/file-utils-easy.test.js]: All the functions on `fue` return Promises

File: test/file-utils-easy.test.js [hunk 4]
```diff
@@ -35,6 +35,32 @@ describe('write tests', () => {
...
+    const filePath = 'append.txt';
+    expect(fue.writeToFile(fileContent, filePath)).resolves.toEqual(filePath);
+    const addingContent = ' - this is some added text';
+    expect(fue.appendToFile(addingContent, filePath)).resolves.toEqual(filePath);
+    return expect(fue.readFile(filePath)).resolves.toEqual(fileContent + addingContent);
+  });
+
+  it('append in not existing path', () => {
+    const fileContent = 'Hello world';
+    const filePath = 'output/hello.txt';
```
Comment (Eomm) [inline test/file-utils-easy.test.js]: For clarity is better to use a `not-exist.txt` path

File: README.md [hunk 5]
```diff
@@ -24,6 +24,7 @@ fue.readDirectoryFiles('path/').then(files => doSomething(files));
 |----------|-------------|
 | `writeToFile`         | Write a string to a file
 | `writeToFileStream`   | Pipe a stream to a file
+| `appendToFile`        | Append a string to a file if already exists, it creates it otherwise
```
Comment (Eomm) [inline README.md:27]: We should be more clear: 
if the path and the file don't exist an error will be thrown
if the path exists and the file doesn't, all will work fine

Comment (niccord) [reply README.md:27]: You didn't specify that for `writeToFile`

File: test/file-utils-easy.test.js (hunk 4 above)
Comment (niccord) [reply test/file-utils-easy.test.js]: I did the same as `write in not existing path` test

File: README.md (hunk 5 above)
Comment (Eomm) [reply README.md:27]: In fact I said "we should" 🤣

File: test/file-utils-easy.test.js (hunk 4 above)
Comment (Eomm) [reply test/file-utils-easy.test.js]: Could you fix also there?
Thanks

**Comment by niccord** [comment]
I am going to fix this

**Pull request closed without merging**